agent/command/git.go

```diff
@@ -158,6 +158,10 @@ func getProjectMethodAndToken(ctx context.Context, comm client.Communicator, td
 	owner := conf.ProjectRef.Owner
 	repo := conf.ProjectRef.Repo
 	appToken, err := comm.CreateInstallationToken(ctx, td, owner, repo)
+	if err == nil {
```

Comment:
Would it be more appropriate to check `appToken` != ""`? Checking `appToken` instead of `err` seems more direct and maybe a tiny bit future-proofed in case the `appToken` is somehow empty and there's no `err`.